### PR TITLE
[REF] i18n: Update translations

### PR DESCRIPTION
**Pull request by hugho-ad**

### CODE REVIEW

**Comment by hugho-ad** [comment]
Red CI no related

**Comment by hugho-ad** [comment]
@oscarolar FYI

File: default_warehouse_from_sale_team/i18n/es_MX.po [hunk 1]
```diff
@@ -4,14 +4,256 @@
...
+
+#. module: default_warehouse_from_sale_team
+#: model:res.groups,name:default_warehouse_from_sale_team.group_manager_default_warehouse_sp
+msgid "Access to all pickings"
+msgstr "Acceso a todos los albaranes"
+
+#. module: default_warehouse_from_sale_team
+#: model:ir.model.fields,help:default_warehouse_from_sale_team.field_crm_team__journal_team_ids
+msgid "Choose the Journals that user with this sale team can see"
+msgstr "Elija las revistas que el usuario con este equipo de venta puede ver"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/i18n/es_MX.po:36]: wey no mames

Comment (hugho-ad) [reply default_warehouse_from_sale_team/i18n/es_MX.po:36]: jajaja, No lo puedo creer jaja

Comment (hugho-ad) [reply default_warehouse_from_sale_team/i18n/es_MX.po:36]: Como diste con esto? esto es de hace un año

Comment (luisg123v) [reply default_warehouse_from_sale_team/i18n/es_MX.po:36]: I noticed the reason some features are not working in v11 is that they were fixed/applied only in v12, so I'm back-porting them on https://github.com/Vauxoo/addons-vauxoo/pull/1416

**Pull request merged**